test/JDBC/expected/parallel_query/babel_collection.out

```diff
@@ -183,16 +183,18 @@ GO
 ~~START~~
 text
 Query Text: select c1 from testing4 where c1 LIKE 'jones'
-Gather  (cost=11.40..24.02 rows=3 width=32)
-  Workers Planned: 3
-  ->  Parallel Bitmap Heap Scan on testing4  (cost=11.40..24.02 rows=1 width=32)
-        Filter: ((c1)::text = 'jones'::text)
-        ->  Bitmap Index Scan on c1_idxtesting49a168d73f3ba5aacdfd495b931b8d187  (cost=0.00..11.40 rows=650 width=0)
+Gather  (cost=4.17..11.28 rows=3 width=32)
```

Comment:
can you please examplain Parallel Bitmap heap scan before vs Bitmap scan currently?

Reply:
There was no Index condition before:
```
 Index Cond: (c1 = 'jones'::"varchar")
```
which means, we were unable to use index efficiently